nix-test/src/const.c

```diff
@@ -209,6 +209,31 @@ get_int_const(const char* err) {
     GET_CONST(ECANCELED);
 #endif
 
+#if defined(__OpenBSD__)
```

Comment:
Grr, I thought we had already removed that.  Go ahead and leave these changes in for now; we'll delete them in a future PR.